launcher/launcher.sh

```diff
@@ -34,6 +34,9 @@ SED_SCRIPT=${TEST_TMPDIR}/runner_config.sed
 # enums.
 echo 's#\\*\"@!([^!]+)!@\\*\"#\1#g' > ${SED_SCRIPT}
 
+unset rlocation
+unset is_absolute
```

Comment:
What's this doing?

Reply:
They're functions that created by bazel during test. The generation of sed script doesn't take care of function, so the script will be formatted incorrectly. The proper fix would be explicitly name which env vars to print in `printenv` or somehow handle functions properly during the generation.

Reply:
Add a comment explaining this, or adjust the grep expression used to remove lines from the printenv output and add the comment there. It'd be very difficult to know ahead of time exactly which environment variables we want to expand.

Reply:
Added comments.